src/components/utils.ts

```diff
@@ -1,3 +1,4 @@
+import { InstallProgress } from './../types';
```

Comment:
```suggestion
import { InstallProgress } from '../types';
```

Reply:
Weird that the automatic import got the wrong file? 🤔

Reply:
Fixed